### DISCUSSION TITLE
Project Submission

### DISCUSSION THREAD

**Discussion by HackerGwen** [github/india#822] [Students]
![Screenshot (458)](https://user-images.githubusercontent.com/98099005/160351622-a8284731-5e1f-4334-8f2d-47d17e2c53b9.png)
I can't find out what I am missing. Please can somebody check that project on my repository? So I can be sure that there is nothing wrong with GitHub?

**Comment by ey-mentor1** [comment]
As per the comments, the GitHub repository link that you had submitted do not exist.
Can you please recheck if the link you are sharing is accessible and valid.

Also, as per mentor comments he/she is not able to identify any azure service in your project.

Hope this resolves your query

**Reply by HackerGwen** [reply]
@ey-mentor1 Thanks for your response but I have checked that their is a workflow document in my GitHub repository that I didn't created. It was generated automatically from Azure. Can you please check it once from given link?

https://github.com/HackerGwen/Azure-Project

**Reply by ey-mentor1** [reply]
Hello,
The link you shared is GitHub domain link.
Since there are no azure dependencies or any azure service used in the given repository, we cannot approve the project.

Can you please host your same project on any azure service and then submit.
Post that, we will evaluate it ASAP. 

Hope this resolves your query regarding your project rejection

Thank you

**Reply by HackerGwen** [reply]
Thanks I'll host that again

**Comment by HackerGwen** [comment]
@ey-mentor1
![Screenshot (463)](https://user-images.githubusercontent.com/98099005/165153084-e61fb784-7470-4a63-a783-bf61990d66b0.png)
![Screenshot (464)](https://user-images.githubusercontent.com/98099005/165153099-de567206-3fb6-453d-8d71-5c3fc69c7b2f.png)
 
I have attached the screenshots of my workflow. Can you please check it again?

**Reply by ey-mentor1** [reply]
Hello,
Yes, this project is new one and no student has shared similar project.

In terms of project synopsis, if you are not able to update t
It on FRT portal, then please mention all the synopsis details in the GitHub readme.
Thank you
